skytemple_randomizer/randomizer/util/util.py

```diff
@@ -196,16 +196,6 @@ def replace_strings(original: str, replacement_map: dict[str, str]):
     return string
 
 
-def replace_text_main(string_file: Str, replace_map: dict[str, str], start_idx, end_idx):
```

Comment:
I've opted to remove this routine from here as nothing else is using it, and the `start_idx`/`end_idx` seems to function as an _exclusion_ instead of range specification, though I'm happy to bring it back if desired. The functionality is now in `npc.py`.